Gemfile

```diff
@@ -17,6 +17,8 @@ else
   gem 'journey'
 end
 
+gem 'i18n', '~> 0.6.11'
```

Comment:
Do we need to enforce this version on the Gemfile? I18n 0.7 should work fine on 3.2 with Ruby 1.9.3+, if it is not we may need to check why, did you get any error?